products/base/{{cookiecutter.product_name}}/config/product.json

```diff
@@ -1,7 +1,17 @@
 {
     "Product" : {
         "Id" : "{{cookiecutter.product_id}}",
-        "Name" : "{{cookiecutter.product_name}}",
-        "Domain" : "{{cookiecutter.domain_id}}"
-    }
-}
\ No newline at end of file
+        "Name" : "{{cookiecutter.product_name}}"{% if cookiecutter.domain_zone != "" %},
+        "Domain" : "{{cookiecutter.product_id}}"{% endif %}
+    }{% if cookiecutter.domain_zone != "" %},
+    "Domains" : {
+        "{{cookiecutter.product_id}}" : {
```

Comment:
@roleyfoley , domains.json used to have "Validation" : "{{cookiecutter.domain_stem}}" element in "Domains" object. Is it needed here in product.json now?

Reply:
```suggestion
        "Validation" : "{{cookiecutter.domain_zone}}",
        "{{cookiecutter.product_id}}" : {
```

Reply:
I'm not sure, I could see it mentioned in CMDB's but couldn't find a trace of it in the engine code 

@ml019 Any ideas on if this Validation field is actively used?

Reply:
Its used in `extendOpenapiDefinition()` by way of the function `getAWSValidation()` - so I'd say its at least checked for in all APIGateways

Reply:
from @ml019 

> This was used when you had to validate ACM certificates using email addresses. This would allow you to override the domain name of the validation email address 
> It can be removed as a default option considering we use DNS validation for ACM Certificates